include/SFML/System/Angle.inl

```diff
@@ -0,0 +1,258 @@
+//
+// 2. Altered source versions must be plainly marked as such,
+//    and must not be misrepresented as being the original software.
+//
+// 3. This notice may not be removed or altered from any source distribution.
+//
+////////////////////////////////////////////////////////////
+
+namespace priv
+{
+    constexpr float pi = 3.141592654f;
+    constexpr float two_pi = 2 * pi;
+
+    constexpr float positiveRemainder(float a, float b)
+    {
+        if (a >= 0.f)
+            return a - static_cast<float>(static_cast<int>(a / b)) * b;
+        else
+            return a + b - static_cast<float>(static_cast<int>(a / b)) * b;
+    }
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle::Angle() :
+m_radians(0.0f)
+{
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr float Angle::asDegrees() const
+{
+    return m_radians * 180 / priv::pi;
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr float Angle::asRadians() const
+{
+    return m_radians;
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle Angle::clampedTo180() const
+{
+    return radians(priv::positiveRemainder(m_radians + priv::pi, priv::two_pi) - priv::pi);
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle Angle::clampedTo360() const
+{
+    return radians(priv::positiveRemainder(m_radians, priv::two_pi));
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle::Angle(float radians) :
+m_radians(radians)
+{
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle degrees(float degrees)
+{
+    return Angle(degrees * priv::pi / 180);
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr Angle radians(float radians)
+{
+    return Angle(radians);
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr bool operator ==(Angle left, Angle right)
+{
+    return left.asRadians() == right.asRadians();
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr bool operator !=(Angle left, Angle right)
+{
+    return left.asRadians() != right.asRadians();
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr bool operator <(Angle left, Angle right)
+{
+    return left.asRadians() < right.asRadians();
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr bool operator >(Angle left, Angle right)
+{
+    return left.asRadians() > right.asRadians();
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr bool operator <=(Angle left, Angle right)
+{
+    return left.asRadians() <= right.asRadians();
+}
+
+
+////////////////////////////////////////////////////////////
+constexpr bool operator >=(Angle left, Angle right)
+{
+    return left.asRadians() <= right.asRadians();
```

Comment:
Wrong operator.

Reply:
Haha, I copy pasted the wrong implementation and wrong tests but they both lined up so I never noticed. Thanks for catching that silly mistake.